Automattic/wp-calypso
# Stats: Removing Clicks module from Stats page

#### Proposed Changes

* Removing `Clicks` widget from the stats page. This PR only removes the widget from the page, not the component or the route for the details page.

#### Testing Instructions


* navigate to any Stats page and verify that `Clicks` widget is not visible on the page

* Before: `Clicks` widget is visible on the Stats page underneath the `Countries` widget
* After: Widget undernet `Countries` not visible.

#### Pre-merge Checklist






Related to #68680
